src/pages/pt-br/guides/styling.mdx

```diff
@@ -94,6 +113,39 @@ const corPlanoFundo = "rgb(24 121 78)";
 
 📚 Veja nossa [página de referência de diretivas](/pt-br/reference/directives-reference/#definevars) para saber mais sobre `define:vars`.
 
+### Passando `class` para um componente filho
+
+No Astro, atributos HTML como `class` não são passados automaticamente para componente filhos.
+
+Ao invés disso, aceite uma prop `class` no componente filho e aplique-a no elemento raiz. Ao fazer desestruturação, você precisa renomeá-la, pois `class` é uma [palavra reservada](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words) no JavaScript.
```

Comment:
We can link to the Portuguese MDN page here:

```suggestion
Ao invés disso, aceite uma prop `class` no componente filho e aplique-a no elemento raiz. Ao fazer desestruturação, você precisa renomeá-la, pois `class` é uma [palavra reservada](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Lexical_grammar#palavras-chave_reservadas_do_ecmascript_2015) no JavaScript.
```

Reply:
Nice catch, thanks for the review!